apt-cacher/ng/client/config/file.sls

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- from tplroot ~ "/ng/map.jinja" import apt_cacher_ng with context %}
+{%- from tplroot ~ "/ng/libtofs.jinja" import files_switch with context %}
+
+{%- if grains['os_family'] == 'Debian' %}
+  {%- if apt_cacher_ng.server_address %}
+
+apt-cacher-ng/config/client_config:
+  file.managed:
+    - name: {{ apt_cacher_ng.client_config }}
+    - user: root
+    - group: root
+    - mode: '644'
+    - source: {{ files_switch(['client.conf.tmpl'],
+                              lookup='apt-cacher-ng/config/client_config'
```

Comment:
```suggestion
                              lookup='apt-cacher-ng/config/client_config',
                              use_subpath=True
```

* That will probably fix the failures in the linked Travis runs.

Reply:
> That will probably fix the failures in the linked Travis runs.

it will. Thanks for the review! I've got a couple of changes to make too.